src/odemis/driver/piezomotor.py

```diff
@@ -511,17 +518,17 @@ def getEncoderPosition(self, axis):
         axname = [name for name, num in self._axis_map.items() if num == axis][0]  # get axis name from number
         return int(self._sendCommand(b'X%dE' % axis)) / self._counts_per_meter[axname]
 
-    def runRelTargetMove(self, axis, encoder_cnts):
+    def runRelTargetMove(self, axis, encoder_cnts, speed):
```

Comment:
I would rename this to speed_steps, because in the rest of the module speed is used for speeds in [m/s] and speed_steps is used for speed in [steps/s]

Reply:
Well, the functions `runAbsTargetMove()` and `runMotorJog()` next to it already have the "speed" argument. So I think it's still clearer to use the same name.